### PR TITLE
Rating: fix focus outline clipping plus updates to line-height and padding

### LINKED ISSUE #6835: Rating component styles are not in concordance with the toolkit
### Bug Report 
 
Rating component is out of design specs when it comes to `line-height` and `padding` around the star. The issue was discovered during fluent work

- __Package version(s)__: latest
- __Browser and OS versions__: all
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? yes
 
Requested priority: Normal
 
#### Describe the issue:
 
When focusing a star the outline clips the left part of it. Also, there is a wrong `line-height` and `padding` on both small and big stars. See the screenshots below. Talked to @betrue-final-final and he approved for changes in the current Rating styles following the fluent toolkit minus the colors. This will also simplify the fluent styles overrides.
 
#### Actual behavior:
screen shot 2018-10-24 at 11 57 30 am

#### Expected behavior:
screen shot 2018-10-24 at 11 58 24 am

### PR DESCRIPTION
**Pull request by Vitalius1**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #6835 
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Changing the line-height and padding on each star to fix focus outline clipping issue as described in the issue addressed in this PR.

The line height matches to fluent redlines after discussing it with design (@betrue-final-final )

#### Focus areas to test

Would this count as minor or patch???

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts [hunk 1]
```diff
@@ -148,22 +151,23 @@ export function getStyles(props: IRatingStyleProps): IRatingStyles {
...
+        lineHeight: ratingLargeIconSize + 'px',
+        height: ratingLargeIconSize + 'px'
       }
     ],
     labelText: [classNames.labelText, hiddenContentStyle],
     ratingFocusZone: [
       classNames.ratingFocusZone,
       {
-        display: 'inline-block',
-        paddingBottom: '1px'
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Rating/Rating.styles.ts]: Removed this 1px padding as it was interfering with the overall height of the Rating wrapper. Have no idea why it even was needed in here.

**Comment by Vitalius1** [comment]
@betrue-final-final there some screener changes with the fix I made here. Would appreciate a review of the changes. Thanks!

File: common/changes/office-ui-fabric-react/v-vibr-RatingRedLinesChange_2018-10-24-20-37.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Rating: Updating padding on each star to fix clipping focus outline and adjust height to follow redlines.",
+      "type": "minor"
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/v-vibr-RatingRedLinesChange_2018-10-24-20-37.json]: To answer your question in your description, this counts as patch :)

**Comment by Vitalius1** [comment]
@cschleiden would appreciate a review on this PR so I can unblock another one. Thanks!

**Pull request merged**